website/example.css

```diff
@@ -0,0 +1,163 @@
+    margin: 0;
+    border: none;
+}
+
+*:focus {
+    outline: none;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------------ */
+
+#header {
+    width: 100%;
+    height: 80px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-image: linear-gradient(to bottom, #38458a, #6a74a1)
+}
+
+#header #logo {
+    padding: 0 50px;
+    font-size: 19px;
+    font-weight: bold;
+    color: white;
+    margin: auto 0;
+}
+
+#header #nav {
+    list-style: none;
+    margin: auto 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+#header #nav li a {
+    color: white;
+    font-size: 16px;
+    font-weight: 500;
+    padding: 0 20px;
+    cursor: pointer;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------------ */
+
+#examples {
+    width: 100%;
+    /* min-height: 400px; */
+    height: auto;
+}
+
+#section-title {
+    font-weight: bold;
+    font-size: 35px;
+    color: #2d3436;
+    padding: 30px 50px 10px 50px;
+}
+
+#section-subtitle {
+    padding: 5px 50px 10px 50px;
+    font-size: 20px;
+    color: #8395a7;
+}
+
+#examples #examples-section {
+    width: auto;
+    height: auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin: 0 30px 0 30px;
+    flex-wrap: wrap;
+}
+
+.eg-card {
+    width: 200px;
+    height: 250px;
+    border-radius: 7px;
+    box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.1);
+    margin: 10px 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+.eg-card .eg-img {
+    margin: 0 auto;
+    height: 60%;
+}
+
+.eg-card .eg-title {
+    font-weight: bold;
+    font-size: 22px;
+    text-align: center;
+    color: #2d3436;
+}
+
+.eg-card:hover {
+    transform: scale(1.05);
+    opacity: 0.75;
+    cursor: pointer;
+    transition: 0.5s;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------------ */
+
+#code-box {
+    width: 100%;
+    min-height: 200px;
+    height: auto;
+    background-color: #f9fbfc;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------------ */
+
+#footer {
+    width: 100%;
+    height: 80px;
+    background-color: #c8d6e5;
+    text-align: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+#footer a {
+    font-size: 30px;
+    color: #8395a7;
+    margin: auto 0;
+    padding: 0 50px;
+    cursor: pointer;
+}
+
+#footer p {
+    font-weight: 500;
+    font-size: 17px;
+    color: white;
+    margin: auto 0;
+    padding: 0 50px;
+}
+
+#footer p i {
+    color: #ff6b6b;
+}
+
+#footer p span {
+    font-weight: 600;
+    color: #10ac84;
+}
```

Comment:
There are a lot of repeated values in here, maybe using [css variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) would be a simpler approach and help with making responsive and additional updates cleaner.